src/js/themes/base.js

```diff
@@ -212,6 +212,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'focus',
         },
       },
+      disabled: {
```

Comment:
I think it might be more consistent to put this under `global.control.disabled`, since that is where the border for TextInput, TextArea, and Select are all defined. And, `disabled` feels more like it should be associated with a control context than with everything.